zookeeper-server/src/main/resources/lib/netty-codec-4.1.105.Final.LICENSE.txt

```diff
@@ -1,7 +1,7 @@
 
                                  Apache License
                            Version 2.0, January 2004
-                        http://www.apache.org/licenses/
+                        https://www.apache.org/licenses/
```

Comment:
Where do these changes come from?

Reply:
It looks like the netty repo includes these changes, from https://github.com/netty/netty/pull/10708

I'm not really sure why we're bundling so many copies of the Apache License anyway. The project as a whole is under the Apache License. Portions of the project, like certain bundled dependencies, may be available under a different license, so it makes sense to include those when that's the case, but when it's Apache, there's not really a need.

Reply:
I agree. As long as we sort it out that "these deps are licensed under the Apache License, 2.0" in LICENSE or NOTICE, we don't need to bundle all these identical copy, like:

* https://github.com/apache/kvrocks/blob/fd33e55d8e6fd9e674eef8711000ab03e6d87d76/NOTICE#L15-L31
* https://github.com/apache/kvrocks/tree/fd33e55d8e6fd9e674eef8711000ab03e6d87d76/licenses